python/tutorials/03-matrix-multiplication.py

```diff
@@ -336,7 +335,9 @@ def test_correctness(M, N, K, in_dtype, out_dtype):
     a = torch.randn((M, K), device='cuda', dtype=torch.float16)
     b = torch.randn((K, N), device='cuda', dtype=torch.float16)
     triton_output = matmul(a, b)
+    triton_relu_output = matmul(a, b, activation="leaky_relu")
     torch_output = torch.matmul(a, b)
+    torch_relu_output = torch.nn.functional.leaky_relu(torch_output)
```

Comment:
What is the purpose of the change here?

Reply:
yeah, @scxiao ,you are right, I don't need temp variable for later compare.

Reply:
My question is: what is the reason of adding the activation function here?

Reply:
This case is for fused matmul. it has activation but not tested in the pytest